packages/foundations-ts-definitions/types/webhook-schema.ts

```diff
@@ -0,0 +1,306 @@
+import {
```

Comment:
So you will need to move this to it's own directory like the traffic-schema. Reason being the types dir gets rm -rf ed by the definitions cron job so only the auto generated types can live in the types directory

Reply:
thought this was the case. Forgot to check!